src/app/header/header.component.ts

```diff
@@ -29,12 +29,18 @@ export class HeaderComponent implements OnInit {
 
   registerPage() {
     // Open dialog box to Register page
-    const openRegister = this.dialog.open(LoginFormRegisterComponent);
+    const openRegister = this.dialog.open(LoginFormRegisterComponent, {
+      height: '370px',
```

Comment:
You can remove the explicit height here, it can calculate that itself and remove those scrollbars.